internal/rancher/provisioning/v1/cluster.go

```diff
@@ -0,0 +1,34 @@
+package v1
```

Comment:
should we start using the namings aligned with upstream: `cluster_types.go` in this case?

Reply:
Probably as a follow-up, this is not a blocker at all and a real nitty